src/BLDC_sm.c

```diff
@@ -315,24 +315,26 @@ void BL_State_Ctrl(void)
 
     if( BL_ARMING == bl_opstate )
     {
-      static const uint16_t ARMING_TIME_100 = 0x08FFu;
+			static const uint16_t ARMING_TIME_TOTAL = 0x0900; // 0x08FFu;
```

Comment:
After reviewing now I can kind of see using 0x08FF for the Arming Time Total  .. if there was a modulus of 0x0100 doing something that you didn't want happening right away on 0x0900. (Just making the comment ... leave as is ;)